rpc/walletrpc/methods.go

```diff
@@ -262,15 +248,46 @@ func SendToAddress(iCmd interface{}, w *wallet.Wallet) (interface{}, error) {
 
 	var amt *types.Amount
 	var err error
-	switch cmd.Coin {
-	case "MEER":
-		amt, err = types.NewAmount(cmd.Amount)
-		amt.Id = types.MEERID
-	case "QIT":
-		amt, err = types.NewAmount(cmd.Amount)
-		amt.Id = types.QITID
-	default:
-		return nil, fmt.Errorf("There is no %s ", cmd.Coin)
+	amt, err = types.NewAmount(cmd.Amount)
+	amt.Id = types.NewCoinID(cmd.Coin)
```

Comment:
Since the new Coin List mechanism is run-time determined, it can only be dynamically obtained by RPC `getTokenInfo` request with the help of a Qitmeer node. Then fill in the result in the local `types.CoinIDList`, so that you can use this interface `types.NewCoinID`